CMakeLists.txt

```diff
@@ -194,20 +217,32 @@ endif()
 # Example binaries
 #============================================================================
 
+if(EMSCRIPTEN)
+    set(EXAMPLE_LIB zlibstatic)
+else()
+    set(EXAMPLE_LIB zlib)
+endif()
```

Comment:
I wonder if it would be nicer to move this variable further up, and then you could simplify things like the 
`if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)` section.

Just an idea.

Reply:
I attempt to keep changes minimal as commit aimed to get pulled back in the original repo and as author is merged this one - my repo is pretty obsolete.